jenkins/agent-maven.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     serviceAccountName: oss-agent
     containers:
     - name: maven
-      image: maven:3.8.6-openjdk-8-slim
+      image: maven:3.8.6-openjdk-11-slim
```

Comment:
not for this PR, but we should probably update this to 3.8.8

Reply:
official docker hub does not support openjdk-11 with maven 3.8.8.
https://hub.docker.com/_/maven/tags?page=&page_size=&ordering=&name=openjdk-11